lightning/src/sign/ecdsa.rs

```diff
@@ -24,11 +24,18 @@ use crate::sign::{ChannelSigner, HTLCDescriptor};
 /// policies in order to be secure. Please refer to the [VLS Policy
 /// Controls](https://gitlab.com/lightning-signer/validating-lightning-signer/-/blob/main/docs/policy-controls.md)
 /// for an example of such policies.
+///
+/// Like [`ChannelSigner`], many of the methods allow errors to be returned to support async
+/// signing. In such cases, the signing operation can be replayed by calling
+/// [`ChannelManager::signer_unblocked`] or [`ChainMonitor::signer_unblocked`] (see individual
+/// method documentation for which method should be called) once the result is ready, at which
+/// point the channel operation will resume.
```

Comment:
I noticed that `ChannelManager::signer_unblocked` docs sound like it needs to be called whenever any of these methods errors. It's clear in these docs which `signer_unblocked` method needs to be called, but there it could be clearer and `{Chain,Channel}Monitor::signer_unblocked` could also benefit from referencing these docs IMO. Doesn't need to happen in this PR though.